### ISSUE TITLE
[Update] php-mongodb 1.8.x

### ISSUE THREAD

**Issue by ShaneOH** [oerdnj/deb.sury.org#1464]
Extension mongodb 1.8.x (https://pecl.php.net/package/mongodb) has been released, with 1.8.0 out as of July and 1.8.1 just out a few days ago.

It seems to be required for the newer releases of the Driver installed via `composer require mongodb/mongodb` in combination with php7.4, e.g.

```
    - mongodb/mongodb 1.7.1 requires ext-mongodb ^1.8 -> the requested PHP extension mongodb has the wrong version (1.7.4) installed.
    - mongodb/mongodb 1.7.0 requires ext-mongodb ^1.8 -> the requested PHP extension mongodb has the wrong version (1.7.4) installed.
```

An update to the `php-mongodb` driver would be greatly appreciated!

**Comment by needz** [comment]
On the same note, please consider adding php-mongodb 1.9 to PHP 8.0, since it's not available in the package and only 1.9 supports PHP 8.0 as pe rhttps://docs.mongodb.com/drivers/php

Check https://github.com/mongodb/mongo-php-driver/releases for PHP driver releases.

**Comment by oerdnj** [comment]
The `php-mongodb` is now at 1.9.0
